torch/fx/graph.py

```diff
@@ -380,11 +380,19 @@ def additional_globals(self) -> List[Tuple[str, Any]]:
     def _gen_python_code(
         self, nodes, root_module: str, namespace: _Namespace, *, verbose: bool = False,
     ) -> PythonCode:
+        from torch.utils._triton import has_triton
+
         free_vars: List[str] = []
         body: List[str] = []
         globals_: Dict[str, Any] = {}
         wrapped_fns: Dict[str, None] = {}
 
+        if has_triton():
+            import triton
```

Comment:
can we lift "import triton" to common file, and only import it once at the beginning?